From 7725942e90c49ea66456242c8b63b13422cf4644 Mon Sep 17 00:00:00 2001
From: Mervin Fansler <mmfansler@gmail.com>
Date: Tue, 6 Aug 2024 22:30:11 +0200
Subject: [PATCH] no pkg-config ucrt

---
 src/Makevars.ucrt | 10 ++--------
 1 file changed, 2 insertions(+), 8 deletions(-)

diff --git a/src/Makevars.ucrt b/src/Makevars.ucrt
index fe7363d..58315b8 100644
--- a/src/Makevars.ucrt
+++ b/src/Makevars.ucrt
@@ -1,11 +1,5 @@
-PKG_CPPFLAGS = -DS2_USE_EXACTFLOAT -D_USE_MATH_DEFINES -DNDEBUG -DIS_LITTLE_ENDIAN -DOMIT_STRPTIME -I../src
-
-ifeq (,$(shell pkg-config --version 2>/dev/null))
-  PKG_LIBS = -Ls2 -ls2static -lssl -lcrypto -lz -lws2_32 -lgdi32 -lcrypt32
-else
-  PKG_LIBS = -Ls2 -ls2static $(shell pkg-config --libs openssl)
-endif
-
+PKG_CPPFLAGS = -I$(LIBRARY_INC) -DS2_USE_EXACTFLOAT -D_USE_MATH_DEFINES -DNDEBUG -DIS_LITTLE_ENDIAN -DOMIT_STRPTIME -I../src
+PKG_LIBS = -L$(LIBRARY_LIB) -Ls2 -ls2static -lssl -lcrypto -lz -lws2_32 -lgdi32 -lcrypt32 -lpthread
 
 CXX_STD = CXX14

